config/genesis.toml

```diff
@@ -16,3 +18,5 @@ auth_seed = "0xd123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef"
 token_symbol = "POL"
 decimals = 12
 max_supply = 1000000
+# To make this account on_chain uncomment the following
+# on_chain = true
```

Comment:
Similar to the previous comments: I would probably call this `storage_mode` (we should probably rename `AccountStorageType` into `AccountStorageMode` - but that's for another PR) and not comment it out. Also, I'd move it right after `type` on line 14.

Reply:
moved this up, uncommented and did the change suggested. Now it's `storage_mode` and it can have the values `on-chain` and `off-chain`. I wasn't able to use `AccountStorageType` straight up because it said it didn't implement Deserialize (though it might be fixable and I didn't find a way), so I ended up writing a `parse_storage_mode`

Reply:
I also updated the test because it was failing and added a check on it asserting that the accounts have the proper storage mode